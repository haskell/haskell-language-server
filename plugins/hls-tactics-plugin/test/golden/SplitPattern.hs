data ADT = One | Two Int | Three | Four Bool ADT | Five

case_split :: ADT -> Int
case_split One = _
case_split (Two i) = _
case_split Three = _
case_split (Four b a) = _
case_split Five = _
