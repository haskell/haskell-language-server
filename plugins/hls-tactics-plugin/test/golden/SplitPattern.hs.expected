data ADT = One | Two Int | Three | Four Bool ADT | Five

case_split :: ADT -> Int
case_split One        = _
case_split (Two i)    = _
case_split Three      = _
case_split (Four b One) = _0
case_split (Four b (Two i)) = _1
case_split (Four b Three) = _2
case_split (Four b (Four b3 a4)) = _3
case_split (Four b Five) = _4
case_split Five       = _
